scheduler/models.py

```diff
@@ -30,6 +30,7 @@ class UserCourseJoinTable(models.Model):
         UserTable,
         on_delete=models.CASCADE,
     )
+    role = models.CharField(max_length=30, default='TA')
```

Comment:
you get the role from the userId which is a UserTable object and holds role information

Reply:
Fixed.